src/reactive/extMethods/where.ts

```diff
@@ -14,18 +14,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
- // notice_end
+// notice_end
 
-import Observable from '../Observable';
-import { Guard } from '../../system';
+import {Observable} from '../Observable';
+import {Guard} from '../../system';
 
-Observable.prototype.where = function(predicate) {
+Observable.prototype.where = function (predicate) {
```

Comment:
could take this time to rename to `filter` to match RxJS, JS arrays

Reply:
I might follow up on that that one. Get this compiling, in TS then do the switch. Need a little task list like on stash